scripts/check-toffee-plum.sh

```diff
@@ -21,39 +23,69 @@ function status_code() {
 
 function failure_check() {
     if [[ $statuscode != 200 ]] && [[ $1 == "Toffee" ]]; then
+        failure_msg_toffee="\n>:red_circle:  <$url| $ENV> is unhealthy"
```

Comment:
```suggestion
        failure_msg_toffee+="\n>:red_circle:  <$url| $ENV> is unhealthy"
```

Seemed += missing here?